test/AutoDiff/SILOptimizer/vjp_and_pullback_inlining.swift

```diff
@@ -40,7 +40,12 @@ func caller_of_simple_vjp() -> Float {
 @_silgen_name("pb_with_control_flow")
 func pb_with_control_flow(_ x: Float) -> Float {
   if (x > 0) {
-    return sin(x) * cos(x)
```

Comment:
I would just ensure we won't have constant `x` here. For example, you could have an argument of `caller_of_pb_with_control_flow` that you will use to evaluate gradient at.

Reply:
Ah nice catch. Fixed.